src/Mutator/MutatorProfile.php

```diff
@@ -0,0 +1,207 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Mutator;
+
+use Infection\Mutator;
+
+class MutatorProfile
```

Comment:
final please

Reply:
We mark almost none of our other classes final. I'm okay with making this final, but it would not be in line with the rest of the code base.